### PR TITLE
🌐 [i18n-KO] Translated `whisper.md` to Korean

### PR DESCRIPTION
**Pull request by nuatmochoi**
# What does this PR do?

Translated the `whisper.md.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

 May you please review this PR? 
@bolizabeth, @nuatmochoi, @heuristicwave, @mjk0618, @keonju2, @harheem, @HongB1, @junejae, @54data, @seank021, @augustinLib, @sronger, @TaeYupNoh, @kj021, @eenzeenee

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR? @sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/model_doc/whisper.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+# Whisper [[whisper]]
+
+## 개요 [[overview]]
+
+Whisper 모델은 Alec Radford, Jong Wook Kim, Tao Xu, Greg Brockman, Christine McLeavey, Ilya Sutskever에 의해 [Robust Speech Recognition via Large-Scale Weak Supervision](https://cdn.openai.com/papers/whisper.pdf)에서 제안되었습니다.
+
+논문의 요약은 다음과 같습니다:
+
+*우리는 인터넷에서 대량의 오디오 전사를 예측하도록 간단히 훈련된 음성 처리 시스템의 성능을 연구합니다. 68만 시간의 다국어 및 다중 작업 지도(multitask supervision)에 확장했을 때, 결과 모델은 표준 벤치마크에 대해 일반화되며, 미세 조정이 필요 없는 제로샷 전송 설정에서 이전의 완전히 지도된 결과와 경쟁할 수 있는 경우가 많습니다. 인간과 비교할 때, 이 모델은 정확도와 견고성에 접근합니다. 우리는 강력한 음성 처리를 위한 추가 작업의 기반이 될 모델과 추론 코드를 공개합니다.*
```
Comment (jungnerd) [inline docs/source/ko/model_doc/whisper.md:25]:
```suggestion
*우리는 인터넷에서 대량의 오디오를 글로 옮긴 것을 예측하도록 간단히 훈련된 음성 처리 시스템의 성능을 연구합니다. 68만 시간의 다국어 및 다중 작업 지도(multitask supervision)에 확장했을 때, 결과 모델은 표준 벤치마크에 잘 일반화되며, 미세 조정이 필요 없는 제로샷 전송 설정에서 이전의 완전히 지도된(fully-supervised) 결과와 경쟁할 수 있는 경우가 많습니다. 사람과 비교하면, 이 모델은 사람의 정확도와 견고성에 근접합니다. 우리는 강력한 음성 처리를 위한 추가 작업의 기반이 될 모델과 추론 코드를 공개합니다.*
```
일부 수정해보았습니다! 번역하기 어려운 문장이 많네요 🤔

File: docs/source/ko/model_doc/whisper.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+
+
+팁:
+
+- 이 모델은 일반적으로 별도의 미세 조정 없이도 잘 작동합니다.
+- 아키텍처는 고전적인 인코더-디코더 아키텍처를 따르며, 이는 추론을 위해 [`~generation.GenerationMixin.generate`] 함수를 사용한다는 의미입니다.
```
Comment (jungnerd) [inline docs/source/ko/model_doc/whisper.md:32]:
```suggestion
- 아키텍처는 고전적인 인코더-디코더 아키텍처를 따르기 때문에, 추론을 위해 [`~generation.GenerationMixin.generate`] 함수를 사용합니다.
```

**Review by jungnerd — COMMENTED** [review]
좋은 번역 감사합니다! 😆 
리뷰해보았으니 검토 부탁드려요!

**Comment by seank021** [comment]
위의 리뷰 내용을 제외하고는, glossary와 문장 요소 등 다 잘 번역되어 있는 것 같습니다! LGTM!!
좋은 번역 감사합니다!

**Review by ArthurZucker — APPROVED** [review]

**Comment by ArthurZucker** [comment]
Did you want to ping someone else for Korean review or should I merge this?

**Comment by nuatmochoi** [comment]
@ArthurZucker 
> Did you want to ping someone else for Korean review or should I merge this?

The review is complete and I just hope to merge this PR.

**Comment by ArthurZucker** [comment]
Thanks for the contribution!

**Pull request merged**